src/celldega/pre/__init__.py

```diff
@@ -537,6 +541,7 @@ def save_landscape_parameters(
     tile_size=1000,
     image_info={},
     image_format='.webp',
+    use_int_index=False,
```

Comment:
I think we should set this to True by default. What do you think?

Reply:
It is set to`True` in the `run_pre_processing.py`. I feel like this way it is safer..